website/docs/r/cbr_rule.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_rule"
+description: |-
+  Manages cbr_rule.
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_rule
+
+Provides a resource for cbr_rule. This allows cbr_rule to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_cbr_rule" "cbr_rule" { 
+	description = "this is an example of rule"
```

Comment:
This example doesnt have Required arguments of resource.. Is this a valid example?

Reply:
updated with a valid example